### PR TITLE
throw exception if size incorrect for ECPoint, UInt160 and UInt256

### PR DESCRIPTION
**Pull request by Qiao-Jin**

### CODE REVIEW

File: src/Neo.SmartContract.Framework/ECPoint.cs [hunk 1]
```diff
@@ -8,15 +8,15 @@ public class ECPoint
         [OpCode(OpCode.DUP)]
         [OpCode(OpCode.SIZE)]
         [OpCode(OpCode.PUSHINT8, "21")] // 0x21 == 33 bytes expected array size
-        [OpCode(OpCode.NUMEQUAL)]
-        [OpCode(OpCode.ASSERT)]
+        [OpCode(OpCode.JMPEQ, "3")]
```
Comment (erikzhang) [inline src/Neo.SmartContract.Framework/ECPoint.cs:11]:
```suggestion
        [OpCode(OpCode.JMPEQ, "03")]
```

File: src/Neo.SmartContract.Framework/ECPoint.cs [hunk 2]
```diff
@@ -8,15 +8,15 @@ public class ECPoint
...
+        [OpCode(OpCode.JMPEQ, "03")]
+        [OpCode(OpCode.THROW)]
         public static extern explicit operator ECPoint(byte[] value);
 
         [OpCode(OpCode.DUP)]
         [OpCode(OpCode.SIZE)]
         [OpCode(OpCode.PUSHINT8, "21")] // 0x21 == 33 bytes expected array size
-        [OpCode(OpCode.NUMEQUAL)]
-        [OpCode(OpCode.ASSERT)]
+        [OpCode(OpCode.JMPEQ, "3")]
```
Comment (shargon) [inline src/Neo.SmartContract.Framework/ECPoint.cs:18]:
```suggestion
        [OpCode(OpCode.JMPEQ, "03")]
```

**Review by shargon — DISMISSED** [review]

**Comment by erikzhang** [comment]
And `UInt160` and `UInt256`?

**Comment by Qiao-Jin** [comment]
> And `UInt160` and `UInt256`?

Updated

**Review by shargon — APPROVED** [review]

**Review by erikzhang — APPROVED** [review]

**Pull request merged**